akka-actor-tests/src/test/scala/akka/event/LoggingReceiveSpec.scala

```diff
@@ -229,10 +230,9 @@ class LoggingReceiveSpec extends WordSpec with BeforeAndAfterAll {
           val sname = supervisor.path.toString
           val sclass = classOf[TestLogActor]
 
-          expectMsgAllPF(messages = 3) {
+          expectMsgAllPF(messages = 2) {
             case Logging.Debug(`sname`, `sclass`, msg: String) if msg startsWith "started" ⇒ 0
             case Logging.Debug(_, _, msg: String) if msg startsWith "now supervising"      ⇒ 1
-            case Logging.Debug(_, _, msg: String) if msg startsWith "now watched by"       ⇒ 2
```

Comment:
in this test there is no watch, the "watch by" is from eventStreamUnsubscriber which might be from previous test case, not sure, but I don't think this test case is about watch